runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -294,6 +299,7 @@ void registerTable(@Nonnull String namespace,
                 break;
             } catch (TransactionAbortedException txAbort) {
                 if (numRetries <= 0) {
+                    log.error("registerTable failed. Retries exhausted. Cause {}", numRetries, txAbort);
```

Comment:
Does it need one more `{}` for `numRetries`?

Reply:
`log.error` takes the error as the last default parameter right?